config/manifests/kustomization.yaml

```diff
@@ -5,6 +5,7 @@ resources:
 - ../default
 - ../samples
 - ../scorecard
+- consoleclidownload/consoleclidownload.yaml
```

Comment:
I would suggest create a directory under `manifest` with the name `shipwright` and include this `consoleclidownload.yaml` inside that along with its own kustomize file.

Then it will copied in `bundle/manifests` folder when `generate bundle` target will be called.